index.html

```diff
@@ -0,0 +1,268 @@
+      </div>
+      <div class="header_item">
+        Pages
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Blog
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
+      <div class="header_item">
+        Contact
+      </div>
+      <div class="header_menu">
+        <img src="images/menu.png" alt="manu-icon" class="header_menu_icon">
+      </div>
+    </div>
+    <!-- ######################################### DARK MODE ######################################### -->
+    <div class="darkmode"><img src="images/moon.png" alt="moon-icon" class="darkmode_icon"></div>
+    <!-- ######################################### BANER SECTION ######################################### -->
+    <div class="container_top">
+      <div class="container_top_background">
+        <img src="images/bubble-21.png" class="bubble21">
+        <img src="images/banner-thumb-4.png" class="banner">
+        <img src="images/line-4.png" class="line4">
+        <img src="images/bubble-9.png" class="bubble9">
+      </div>
+      <div class="container_top_conent">
+        <div class="baner_text">
+          <div class="baner_title">Design</div>
+          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum.
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### ABOUT SECTION ######################################### -->
+    <div class="container_about">
+      <div class="left">
+        <h7>About Us</h7>
+        <div class="aboutUs_title">Why branding matters?</div>
+        <div class="about_text">
+          It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
+        </div>
+        <div class="list_select">
+          <img src="images/strategy.png" class="strategy">
+          Strategy
+          <div class="list_text">Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model and web page editors</div>
+        </div>
+        <div class="list">
+          <img src="images/design.png" class="design">
+          Design
+        </div>
+        <div class="list">
+          <img src="images/earth.png" class="earth">
+          Development
+        </div>
+      </div>
+      <div class="right">
+        <div class="form_container">
+          <div class="form_title">Get a free Keystroke <br /> quote now</div>
+          <div class="form_name">Name</div>
+          <div class="form_inbox">John Smith</div>
+          <div class="form_name">Email</div>
+          <div class="form_inbox">example@mail.com</div>
+          <div class="form_name">Phone</div>
+          <div class="form_inbox">+123456789</div>
+          <div class="form_button">Get Pricing Now</div>
```

Comment:
Sama nazwa klasy wskazuje że button.

```suggestion
          <button class="form_button">Get Pricing Now</button>
```